### PR TITLE
Fix IE8 errors. Closes #45

### PR DESCRIPTION
**Pull request by wnr**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,6 +10,36 @@ module.exports = function(css, options){
...
+  function trim(str) {
+    if (str === void 0 || str === null) {
+      throw new TypeError('trim called on null or undefined');
+    }
+
+    var ws = '\x09\x0A\x0B\x0C\x0D\x20\xA0\u1680\u180E\u2000\u2001\u2002\u2003' +
+    '\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u202F\u205F\u3000\u2028' +
+    '\u2029\uFEFF';
+
+    if (!String.prototype.trim || ws.trim()) {
```
Comment (tj) [inline index.js]: hmm this seems a bit extreme, I'd just do `str.replace(/^\s+|\s+$/g, '')` with no fallback on the native one, regexps are plenty fast

Comment (wnr) [inline index.js]: Okay :) Shall I fix this or will you do it?

Comment (wnr) [inline index.js]: I replaced it now with the regex given. To not get a function call overhead I didn't wrap the regex in a trim function.

**Comment by tj** [comment]
I wouldn't worry about the function overhead, you can typically invoke a function ~5,000,000+ times a second which is plenty haha but good enough :D  thanks!

**Comment by wnr** [comment]
Haha okay! I guess I let some SO answer impact me too much regarding function overheads :p 
Glad to help :)

**Pull request merged**